java/drivers/driver-hazelcast4plus/pom.xml

```diff
@@ -17,7 +17,7 @@
     </prerequisites>
 
     <properties>
-        <hazelcast.version>5.2.1</hazelcast.version>
+        <hazelcast.version>5.3.1</hazelcast.version>
```

Comment:
in #2036 @Fly-Style used 5.3.0 - what it the correct value?

Reply:
Not sure. I was just updating to the latest stable version. If we decide that 5.3.0 is best/correct it shouldn't cause any problems.

Reply:
@k-jamroz Let's use the latest stable version.

Reply:
Updated to 5.3.2 :+1: